pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+env =
+	FLASK_ENV=testing
```

Comment:
Does uhh, does this have tabs? 😅 Should I just stop worrying about it? Lol

Reply:
Oh no! I swear I didn't mean to 😭